### PR TITLE
chore: overwrite the default console.error in logQueryError

### PR DESCRIPTION
**Pull request by cyjake**
make sure server disk does not get flooded with leoric logs by default

### CODE REVIEW

**Review by JimmyDaddy — APPROVED** [review]
LGTM

File: config/config.default.js [hunk 1]
```diff
@@ -1,5 +1,7 @@
 'use strict';
 
+const debug = require('debug')('egg-orm');
```
Comment (yisibl) [inline config/config.default.js:3]: 少了 `)`?

Comment (cyjake) [reply config/config.default.js:3]: 又看了一遍，没少啊

Comment (yisibl) [reply config/config.default.js:3]: Sorry，可能昨天 GitHub bug 了

**Pull request merged**